rust/windows-client/src-tauri/src/client/gui.rs

```diff
@@ -227,53 +227,17 @@ fn handle_system_tray_event(app: &tauri::AppHandle, event: TrayMenuEvent) -> Res
         .ok_or_else(|| anyhow!("can't get Managed struct from Tauri"))?
```

Comment:
Did you know that you can use `.context` also on `Option`s to turn them into `Result`s? Would be a bit simpler in this case :)

Reply:
I think I didn't know that 2 months ago, thanks :)